docs/intro/getting-started.rst

```diff
@@ -16,6 +16,12 @@ The first step is to create a project directory, and clone Batavia:
     $ cd tutorial
     $ git clone https://github.com/pybee/batavia.git
 
+Batavia now requires a copy of Ouroboros (the Python standard library, written in Python) to build, so we also need to clone that.
```

Comment:
No need for the tense here - someone coming to the project for the first time won't have the context of "before".